components/centraldashboard/public/components/namespace-selector.js

```diff
@@ -13,6 +17,36 @@ export class NamespaceSelector extends PolymerElement {
     static get template() {
         return html`
             <style>
+                paper-menu-button {
+                    --paper-menu-button: {
+                        font-size: 14px;
+                        color: #3c4043
+                    }
+                }
+
+                paper-button {
```

Comment:
Polymer tries to promote using id's since DOM level scoping and optimizations are already applied, and makes the CSS more specific while increasing readabilty (ie. what does this paper-button do?)

Reply:
Understood, I've changed it to dropdown-trigger and used the ID for styling purposes. I think I've gotten into the habit of relying on the fact that the template only has a single element of the type in the shadow DOM but it makes sense to be explicit where possible.